lib/active_record/acts_as/relation.rb

```diff
@@ -83,10 +83,13 @@ def is_a?(klass)
           super || acting_as?(klass)
         end
 
-        def actable(options = {})
+        def actable(scope = nil, **options)
```

Comment:
Ensure that any code in coursemology that uses `actable` passes the correct arguments.

Reply:
yes. I checked already our code, and I can assure that this won't break any existing piece of code that calls actable. This one is merely adding the flexibility for us should we define the scoping of actable but if we don't have any scoping defined, it's entirely fine